### PR TITLE
Install and import fixes

### PR DESCRIPTION
**Pull request by sarnold**
Two "minor" fixes to get plugins working and banish import errors.

### CODE REVIEW

File: xl/settings.py [hunk 1]
```diff
@@ -37,7 +37,7 @@
 import logging
 import os
 import sys
-
+import dbus
```
Comment (virtuald) [inline xl/settings.py]: Hm? Settings shouldn't depend on dbus.

**Comment by sarnold** [comment]
It coughs a missing dbus warning starting up in a console; but I couldn't really find a better place.

**Comment by virtuald** [comment]
There is no dbus module in windows, so importing it in settings is probably a bad idea.

**Comment by sarnold** [comment]
I think it was a side-effect of the missing pylast, seems like the install fix took care of it (since I can't reproduce it now after commenting that out in my test package).  Nevermind...

**Comment by sarnold** [comment]
Okay, bye-bye dbus commit...

**Pull request merged**